src/cider/exec/nextgen/parsers/RelAlgEUParser.cpp

```diff
@@ -96,6 +96,9 @@ class InputAnalyzer {
     } else {
       auto children = (*curr)->get_children_reference();
       for (ExprPtr* child : children) {
+        if (!child || !child->get()) {
```

Comment:
in like expr there are 3 arguments: the input, the 'like' pattern and an escape character. Currently the escape character is nullptr (because substrait does not support escape)